docs/docs/user-setup.md

```diff
@@ -33,6 +33,12 @@ Following the steps in this guide, you will:
 1. Update the configuration flags for your user config, to enable/disable any ZMK features you would like to include.
 1. Commit and push the updates to trigger a new GitHub Action run.
 
+:::note
+If you are building firmware for a new keyboard shield that is not included in the built-in
+list of shields, you can choose any shield from the list that is similar to yours to generate the repository, 
+and edit / add necessary files according to the [guide for adding new keyboard shield](./dev-guide-new-shield.md).
+:::
+
```

Comment:
Would this be better placed right next to the step where a user selects their shield? https://zmkfirmware.dev/docs/user-setup/#keyboard-shield-selection